metis-indexing/src/main/java/eu/europeana/indexing/tiers/model/MediaTier.java

```diff
@@ -1,4 +1,4 @@
-package eu.europeana.indexing.solr.tiers.model;
+package eu.europeana.indexing.tiers.model;
 
 /**
  * This enum contains the different values of the media tier.
```

Comment:
Why is the label field required?

Reply:
I anticipated their use, but I saw I now have them also in QualityAnnotationSolrCreator, where they belong. I will remove them.